client/src/extension.ts

```diff
@@ -74,7 +74,35 @@ async function disableInteferingSettings (): Promise<void> {
   }
 }
 
+async function installExtensions (extensionId: string): Promise<void> {
+  await vscode.window.withProgress({
+    location: vscode.ProgressLocation.Notification,
+    title: ` yocto-project.yocto-bitbake depends on extension ${extensionId}, installing it...`,
+    cancellable: false
+  }, async (progress, token) => {
+    try {
+      await vscode.commands.executeCommand('workbench.extensions.installExtension', extensionId).then(() => {

```

Comment:
We should ask for permission before installing them. First show a notification with install/cancell.

Reply:
Then, what do we do when users cancel the installation? Do we continue to remind them of it? Maybe on every startup of a new session.

Reply:
If the user refuses the install we can cancel activation and request next session.

Reply:
Please check the new commit